demos/simple-menu.html

```diff
@@ -28,33 +28,28 @@
       html,
       body,
       main {
-        display: flex;
+        position: relative;
         flex-direction: column;
-        height: 100%;
-      }
-
-      .demo-header-toolbar {
-        z-index: 10;
```

Comment:
Well, true. However, popup menus should show over the title bar. I would argue that z-indexing of the bar itself is not right.

Reply:
I guess you could argue for changing the z-index of the specific hero menu instance instead?  One way or another, this menu shouldn't overlap the toolbar; it breaks the perception of the toolbar being "over" the page.

For future thought: I'm not sure how typical it is for pages to scroll while menus are open, but it seems more common to have a menu within the body of a page rather than in its fixed toolbar and in that case you wouldn't want the menu to cover the toolbar, so ultimately menu-in-toolbar may warrant a special-case `z-index` of its own instead.

Reply:
Ok I put back z-index on the toolbar and added z-index for the popup menu. I still think it sets a bad example moving forward - "z-index war" is a loosing war.